packages/loader/test/index.test.js

```diff
@@ -51,4 +51,4 @@ const testFixture = (fixture, options = {}) => {
 
 test('it loads markdown and returns a component', async () => {
   await testFixture('fixture.md')
-})
+}, 10000)
```

Comment:
Previously Node v8 tests were failing, but now Node v10 were failing too, or instead. This seems to fix it.